source/internal/_isTransformer.js

```diff
@@ -1,3 +1,5 @@
 export default function _isTransformer(obj) {
-  return typeof obj['@@transducer/step'] === 'function';
+  return obj !== null &&
+         typeof obj !== 'undefined' &&
```

Comment:
could use shorthand `obj != null` here

Reply:
I had that first, then switched to the more explicit checks. I'll go with whatever you prefer.

Reply:
My reasoning: I try to avoid depending on magic coercion, but I'm happy to stick to house style here, whatever that may be.

Reply:
i believe that `==` is house style in the `null || undefined` case. I am sympathetic with you wanting to be explicit.

Reply:
:+1: for `!= null`

Reply:
I agree that this is consistent.  I'm not sure how or why we settled on this style as I would also prefer to be explicit.   But I don't find it a big deal.